src/CBT.NuGet/Tasks/GenerateNuGetProperties.cs

```diff
@@ -47,8 +47,6 @@ public GenerateNuGetProperties()
                 //
                 _assemblySearchPaths.Add(Path.GetDirectoryName(AppDomain.CurrentDomain.GetData("CBT_NUGET_ASSEMBLY_PATH").ToString()));
```

Comment:
I think you can remove the `_assemblySearchPaths` and `_loadedAssemblies` variables as well right?  And most of the code in the constructor.